pontoon/localizations/templates/localizations/localization.html

```diff
@@ -18,6 +18,7 @@
 {% block heading %}
 <section id="heading">
   <div class="container clearfix">
+      {{ HeadingInfo.download_button(locale.code, project.slug) }}
```

Comment:
Nit: Please dedent by 2 spaces.

Reply:
I will specially take care of indentation in further bugs.(Noted)